.github/workflows/checkov.yml

```diff
@@ -1,20 +1,22 @@
-name: "checkov"
+name: "Checkov Repository Workflow"
```

Comment:
Please update file to match the [example](https://github.com/honestbank/workflows/blob/main/examples/repository-workflows/repository-checkov.yaml).  Workflow name, job id and job name must all match.

Reply:
@MXfive  this is public repo so it we used the shared workflow then it is failing @honestsubhadip has already done the testing and it was continuously failing Hence, we need to remove that. @honestsubhadip  has created backlog ticket for the same.